### PR TITLE
Static build of ffmpeg

### LINKED ISSUE #1245: Embed FFMPEG as VIAME missing fallback
**Is your feature request related to a problem? If so, Please describe.**
Users who do not have VIAME and are either unwilling or unable to install ffmpeg independently of DIVE should be able to use DIVE as a standalone binary with full annotation features.

**Describe the solution you'd like**

Try embedding https://www.npmjs.com/package/ffmpeg-static

Remove logic to try and use a local `PATH` install, because that's just too much uncertainty for no reason.

**Describe alternatives you've considered**

* A function that downloads ffmpeg at runtime.

### PR DESCRIPTION
**Pull request by subdavis**
* Add `ffmpeg` and `ffprobe` static.
* Remove platform-specific media encoders, just use `spawn` directly without shell.  Remove logic to attempt to use VIAME installation of ffmpeg.
* Remove electron devtools vue (It seems somewhat permanently broken).
* Add handler for multi-instance launch.

fixes #1245

### CODE REVIEW

**Review by BryonLewis — CHANGES_REQUESTED** [review]
I'm not nearly done with a complete review.  I just wanted to add this initially to see if we could get building for windows through the workflow going as well.  I want to take a build and put it through it's paces:

- mis-alignment checking (ffprobe) and see if it does the double transcoding
- AV1 and other weird codecs
- Removing ffmpeg from by PATH and ensuring it still works (my windows testing makes me think it should)
- Image conversion (remember we convert bmps and tifs to pngs I believe)

**Review by BryonLewis — CHANGES_REQUESTED** [review]
All my testing seemed to work perfectly fine.  Only one question about leaving the shell argument in the `spawnResult`.
I'm going to double check the building on Windows today to see if that works well enough.

File: client/platform/desktop/backend/native/mediaJobs.ts [hunk 1]
```diff
@@ -39,7 +69,7 @@ async function checkFrameMisalignment(
     '-v',
     'quiet',
   ];
-  const result = await spawnResult(command.join(' '), viameConstants.shell);
+  const result = await spawnResult(ffprobePath, false, args);
```
[the remark this answers is not in the document]
Comment (subdavis) [reply client/platform/desktop/backend/native/mediaJobs.ts:72]: Yeah, I didn't catch that.  Thanks.  I thought viame stuff used it too.

**Comment by subdavis** [comment]
@BryonLewis I think I found a simpler fix for windows.  I just added a wildcard filter to make sure the `.exe` gets bundled.  Could you test again?  Thanks.

**Review by BryonLewis — APPROVED** [review]
Builds on windows and I could do transcoding.

**Pull request merged**